src/SkyApm.Diagnostics.MassTransit/Common/RabbitmqComponentIdChecker.cs

```diff
@@ -0,0 +1,32 @@
+﻿using SkyApm.Common;
+
+namespace SkyApm.Diagnostics.MassTransit.Common
+{
+    public class RabbitmqComponentIdChecker : IComponentIdChecker
+    {
+        public string PublishEndpointName => "rabbitmq";
+
+        public string ConsumeEndpointName => PublishEndpointName;
+
+        public StringOrIntValue CheckPublishComponentID(string host)
+        {
+            if (ContainsRabbitmq(host))
+                return 52; //rabbitmq-producer 
+            return 51; //rabbitmq
```

Comment:
what is the definition of these numbers ?

Reply:
As [component-libraries.yml](https://github.com/apache/skywalking/blob/master/oap-server/server-starter/src/main/resources/component-libraries.yml) in Skywalking mentioned, each component has it's int value, and it can be regonized as the `rabbitmq-producer`  `rabbitmq-consumer` and so on.

I will need add more IComponentIdChecker to regonize the detail marker.